core/src/main/java/org/apache/airavata/mft/core/api/ConnectorConfig.java

```diff
@@ -90,6 +90,40 @@ public void setTransportConfig(Map transportConfig) {
         this.transportConfig = transportConfig;
     }
 
+    public static final class LocalConfigs {
+        public LocalConfigs() {
+        }
+
+        public final static String DMA_ENABLED = "local.dma";
+        public final static String BUFF_LEN = "local.buffLen";
+    }
+
+    public Boolean getBooleanTransportProperty(String key, Boolean defaultValue){
+        if(key == LocalConfigs.DMA_ENABLED){
```

Comment:
Why do you have a check like this?

Reply:
If we want any further checks in future depending on the key type, we can maintain with this if clause. If not we can just remove and keep it simply same for all.